src/components/organisms/WhyRightOn.jsx

```diff
@@ -9,13 +9,18 @@ import RightOnVideo from '../atoms/RightOnVideo';
 
 function WhyRightOn(props) {
   const { whyRightOn } = props
+
   return (
     <Box component="section">
       <Grid container justify="center" alignItems="center" spacing={5}>
-        <Grid item sm={12}>
-          <SectionHeading />
+        <Grid item sm={12} style={{ zIndex:2 }}>
+          <SectionHeading title="Check out how teachers are using RightOn!" />
         </Grid>
             <RightOnVideo/>
+        <Grid item sm={12} style={{ zIndex:2 }}>
+          <SectionHeading title="Why RightOn"style={{ zIndex:2 }}/>
```

Comment:
Should there be a question mark in the title?

Reply:
There was supposed to me an exclamation mark which has been fixed!